Polyfills/WebSocket/Source/WebSocket.cpp

```diff
@@ -182,13 +182,15 @@ namespace Babylon::Polyfills::Internal
         });
     };
 
-    void WebSocket::CloseCallback()
+    void WebSocket::CloseCallback(int code, const std::string& reason)
     {
-        m_runtimeScheduler([this]() {
+        m_runtimeScheduler([this, code, reason = std::move(reason)]() {
```

Comment:
Unfortunately, this can't be moved because it's const. This has to copy here. Multiple places.
```suggestion
        m_runtimeScheduler([this, code, reason]() {
```